syncd/FlexCounter.cpp

```diff
@@ -1093,6 +1093,38 @@ void FlexCounter::removeDataFromCountersDB(
     }
 }
 
+string FlexCounter::getRifCounterTableKey(string key)
+{
+    return "COUNTERS:" + key;
+}
+
+string FlexCounter::getRifRateTableKey(string key)
+{
+    return "RATES:" + key;
+}
+
+string FlexCounter::getRifRateInitTableKey(string key)
+{
+    return "RATES:" + key + ":RIF";
+}
+
+void FlexCounter::cleanUpRifFromCounterDb(_In_ sai_object_id_t rifId)
```

Comment:
Could you please try reuse removeDataFromCountersDB?

Reply:
@Junchao-Mellanox , Sure , I will use as suggested and check/UT . If it works fine, will submit a patch.